thinkbox-deadline/azuredeploy.json

```diff
@@ -56,10 +65,29 @@
             }
         },
 
+        "repositorySize":

```

Comment:
For clarity sake, it'd be better to call this repositoryVMSize so the user knows what to input while deploying without looking at the template